packages/flutter/lib/src/material/material.dart

```diff
@@ -87,13 +89,15 @@ abstract class MaterialInkController {
 ///
 /// The Material widget is responsible for:
 ///
-/// 1. Clipping: Material clips its widget sub-tree to the shape specified by
-///    [shape], [type], and [borderRadius].
-/// 2. Elevation: Material elevates its widget sub-tree on the Z axis by
+/// 1. Elevation: Material elevates its widget sub-tree on the Z axis by
 ///    [elevation] pixels, and draws the appropriate shadow.
-/// 3. Ink effects: Material shows ink effects implemented by [InkFeature]s
+/// 2. Ink effects: Material shows ink effects implemented by [InkFeature]s
 ///    like [InkSplash] and [InkHighlight] below its children.
```

Comment:
I think we should probably leave the list as it was, but make it clear the clipping is only if requested, just like the shadow is only if requested, and the ink effects are only if requested.